tests/snippets/append.py

```diff
@@ -1,2 +0,0 @@
-x = []
```

Comment:
Why did you remove this file? I agree the test makes little sense, but still, it is a valid test.

Reply:
Because other tests in `list.py` do the same job. No point testing the same thing twice.